connect-example/src/main/java/com/stripe/android/connect/example/ui/common/BasicExampleComponentActivity.kt

```diff
@@ -117,8 +125,15 @@ abstract class BasicExampleComponentActivity : FragmentActivity() {
                     openSettings = openSettings,
                     reload = viewModel::reload,
                 ) { embeddedComponentManager ->
-                    AndroidView(modifier = Modifier.fillMaxSize(), factory = { context ->
-                        createComponentView(context, embeddedComponentManager)
+                    val context = LocalContext.current
+                    LaunchedEffect(context) {
+                        val appearanceInfo = settingsService.getAppearanceId()
+                            ?.let { AppearanceInfo.getAppearance(it, context).appearance }
+                            ?: return@LaunchedEffect
+                        embeddedComponentManager.update(appearanceInfo)
```

Comment:
This is not ideal because in our Example app `EmbeddedComponentManager` is scoped to our Application while we're updating the appearance per Activity. Practically speaking, it's OK, but it doesn't serve as a great example for integrators.